### PR TITLE
[CLEANUP] Consistency fixes

### PR DESCRIPTION
**Pull request by jasonly**
Added commas after usernames there were some inconsistencies between usernames having them and not having them.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -27,18 +27,18 @@ And here are the volunteers so far:
...
+- @anbuselvan, https://github.com/anbuselvan - contributed code for the foundations of the app, including hangout/learning creation and Slack authentication.
+- @husam, https://github.com/hudat
+- @ccr, https://github.com/Iccr
+- @ghazi, https://github.com/ghmoha
+- @mualth, https://github.com/maudem
+- @oliver84, https://github.com/Oliver84
 - @selfspunwebs, https://github.com/anonRegions
-- @luxis https://github.com/Luxisapex - contributed to various issues, including the "kudos" feature which lets users upvote learnings
+- @luxis, https://github.com/Luxisapex - contributed to various issues, including the "kudos" feature which lets users upvote learnings
 - @sergeant-q https://github.com/sergeant-q - contributed fixes for issues, including hangout deletion and email notification for cancelled hangouts, and improved thinking on process steps for how to collaborate on this app
```
Comment (wuworkshop) [inline readme.md]: @jasonly thanks for cleaning up the names! looks like line 42 (sergeant-q's name) is still missing a comma.

Comment (jasonly) [inline readme.md]: Oh sweet! Thanks for the catch!

**Comment by camilleriluke** [comment]
Nice one @jasonly :)

**Comment by lpatmo** [comment]
Thanks @jasonly! :)

**Pull request merged**